test/sys/test_mman.rs

```diff
@@ -1,16 +1,14 @@
-use nix::Error;
-use nix::libc::{c_void, size_t};
 use nix::sys::mman::{mmap, MapFlags, ProtFlags};
 
-#[cfg(target_os = "linux")]
+#[cfg(any(target_os = "linux", target_os = "netbsd"))]
 use nix::sys::mman::{mremap, MRemapFlags};
 
 #[test]
 fn test_mmap_anonymous() {
     let ref mut byte = unsafe {
         let ptr = mmap(std::ptr::null_mut(), 1,
                        ProtFlags::PROT_READ | ProtFlags::PROT_WRITE,
-                       MapFlags::MAP_PRIVATE | MapFlags::MAP_ANONYMOUS, -1, 0)
+                       MapFlags::MAP_PRIVATE | MapFlags::MAP_ANON, -1, 0)
```

Comment:
Was there a need for this change or do you just prefer the shorter symbol?

Reply:
The reason of the change was, `MAP_ANONYMOUS` is defined on only specific target (`android,linux,freebsd`) while `MAP_ANON` is defined for every os.